synapse/config/tls.py

```diff
@@ -40,7 +41,9 @@ def read_config(self, config):
         self.acme_reprovision_threshold = acme_config.get("reprovision_threshold", 30)
 
         self.tls_certificate_file = os.path.abspath(config.get("tls_certificate_path"))
-        self.tls_private_key_file = os.path.abspath(config.get("tls_private_key_path"))
+        self.tls_private_key_file = config.get("tls_private_key_path")
+        if self.tls_private_key_file:
```

Comment:
there's a `self.abspath` to do this boilerplate for you

Reply:
done